.golangci.yml

```diff
@@ -0,0 +1,161 @@
+run:
+  # default concurrency is a available CPU number
+  concurrency: 4
```

Comment:
Why this limit? I thought their sales pitch was that they'd use N-1 processors automatically.

Reply:
It's just their recommendation... I believe there is an issue with incorrect CPU_NUM reporting within the containers. I think circleci runs on machines with 16+ cores and reports as such, yet containers are restricted to 4 cores.